rust/src/accuracy/polars/mod.rs

```diff
@@ -178,6 +190,10 @@ fn expr_utility<'a>(
         }]);
     }
 
+    if let Some((inputs, _)) = match_trusted_plugin::<IndexCandidatesPlugin>(&expr)? {
+        return summarize_expr(&inputs[0], alpha, threshold);
+    }
```

Comment:
This adjustment makes quantile statistics summarize properly.

Reply:
```suggestion
    # summarize quantile statistics properly
    if let Some((inputs, _)) = match_trusted_plugin::<IndexCandidatesPlugin>(&expr)? {
        return summarize_expr(&inputs[0], alpha, threshold);
    }
```